app-modules/stock-media/src/Filament/Pages/ManageStockMediaSettings.php

```diff
@@ -0,0 +1,86 @@
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\StockMedia\Filament\Pages;
+
+use AdvisingApp\StockMedia\Enums\StockMediaProvider;
+use AdvisingApp\StockMedia\Settings\StockMediaSettings;
+use App\Features\StockMediaFeature;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\TextInput;
+use Filament\Forms\Components\Toggle;
+use Filament\Forms\Form;
+use Filament\Forms\Get;
+use Filament\Pages\SettingsPage;
+
+class ManageStockMediaSettings extends SettingsPage
+{
+    protected static ?string $navigationGroup = 'Global Administration';
+
+    protected static ?int $navigationSort = 80;
+
+    protected static ?string $navigationLabel = 'Stock Media Settings';
+
+    protected static string $settings = StockMediaSettings::class;
+
+    public static function canAccess(): bool
+    {
+        return StockMediaFeature::active() && auth()->user()->isSuperAdmin();
+    }
+
+    public function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Toggle::make('is_active')
+                    ->label('Active')
+                    ->inline(false)
+                    ->live(),
+                Select::make('provider')
+                    ->options(StockMediaProvider::class)
+                    ->enum(StockMediaProvider::class)
+                    ->visible(fn (Get $get) => $get('is_active'))
+                    ->afterStateHydrated(fn (Select $component) => $component->state(StockMediaProvider::Pexels))
+                    ->disabled(true)
+                    ->dehydrated(),
+                TextInput::make('pexels_api_key')
+                    ->label('API Key')
+                    ->visible(fn (Get $get) => $get('is_active') && $get('provider') === StockMediaProvider::Pexels)
```

Comment:
I think you also need to account for `$get('provider')` to be the `StockMediaProvider::Pexels->value` as well as `StockMediaProvider::Pexels`, as that can happen when the enum select is used